### PR TITLE
js-2/Week-4/MinKo

### PR DESCRIPTION
**Pull request by minwwgo**

### CODE REVIEW

**Comment by kiowd** [comment]
`.forEach(log)`

File: week-4/Homework/mandatory/2-writers.js [hunk 1]
```diff
@@ -42,3 +42,4 @@ let writers = [
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+let writerAlive= writers.filter(n=>n.alive === true).forEach(n=> console.log(`Hi, my name is ${n.firstName} ${n.lastName}. I am ${n.age} years old, and work as a ${n.occupation}."`))
```
Comment (BexB1) [inline week-4/Homework/mandatory/2-writers.js]: Excellent use of `.filter` and `.forEach`, and well done on completing the extra challenge.

File: week-4/Homework/mandatory/4-groceries.js [hunk 2]
```diff
@@ -3,10 +3,15 @@
...
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange Juice",
+  item3: "Rice"
 };
+for (const property in groceryList){
```
Comment (BexB1) [inline week-4/Homework/mandatory/4-groceries.js]: Good use of a `for` loop. Perhaps you could have used the Object methods we learned last week to update the values in `groceryList`?

File: week-4/InClass/A-objects-intro/exercise-part-2.js [hunk 3]
```diff
@@ -5,17 +5,18 @@ The objects below have some syntax issues - try and fix them all!
 */
 
 let kitten = {
-    fur colour: "orange",
-    age "23"
+    furcolour: "orange",
```
Comment (BexB1) [inline week-4/InClass/A-objects-intro/exercise-part-2.js]: Don't forget to camelCase in JavaScript! `'furcolour' = 'furColour"`

File: week-4/InClass/B-objects-get-set/exercise-2.js [hunk 4]
```diff
@@ -5,14 +5,15 @@
 */
 
 let phone = {
-    brand: 'iPhone,
-    model 'iPhone X'
+    brand: 'iPhone',
+    model :'iPhone X',
```
Comment (BexB1) [inline week-4/InClass/B-objects-get-set/exercise-2.js]: Watch the space between the key/property and the colon here. This line should read `model: 'iPhone X',`

File: week-4/InClass/C-more-complex-objects/exercise-3.js [hunk 5]
```diff
@@ -32,17 +32,18 @@ let parkAvenueHouse = {
...
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return [`${house1.currentOwner.email} , ${house2.currentOwner.email}`]
 }
 
 // returns the address for the cheapest house out of the two
 function getCheapestAddress(house1, house2) {
-
+  return house1.price<house2.price? house1.address:house2.address
```
Comment (BexB1) [inline week-4/InClass/C-more-complex-objects/exercise-3.js]: Good use of the ternary operator, but remember your spacing between the syntax. It keeps things more readable.

**Pull request closed without merging**